explorer/interpreter/action.h

```diff
@@ -58,8 +60,13 @@ class RuntimeScope {
   auto Get(ValueNodeView value_node) const
       -> std::optional<Nonnull<const LValue*>>;
 
+  auto Locals() const -> std::vector<Nonnull<const LValue*>> {
+    return local_values_;
+  }
+
  private:
-  std::map<ValueNodeView, Nonnull<const LValue*>> locals_;
+  std::vector<Nonnull<const LValue*>> local_values_;
+  std::map<ValueNodeView, unsigned int> locals_;
```

Comment:
Consider instead using [`llvm::MapVector`](https://llvm.org/doxygen/classllvm_1_1MapVector.html) -- that gives you a map-like API, while also letting you iterate through the map elements in a fixed order.